tensorflow/MLDB-2088-tensorflow-GPU-support.js

```diff
@@ -0,0 +1,228 @@
+// The URI from which we load the Inception model.  We're assuming
+// that you've already downloaded it from Google at the following
+// URL; you can also load it directly from Google if you have
+// access to a lot of bandwidth by replacing the file:// URL below
+// with the following:
+// https://storage.googleapis.com/download.tensorflow.org/models/inception_dec_2015.zip
+
+var res = mldb.query("SELECT tf_Cos(1.23, {T: { type: 'DT_DOUBLE'}}) AS res");
+
+mldb.log(res);
+
+var res = mldb.query("SELECT tf_MatrixInverse([[1,2,3],[4,-5,6],[-7,8,9]], { T: { type: 'DT_DOUBLE' } }) AS res");
+
+mldb.log(res);
+
+var inceptionUrl = 'file://inception_dec_2015.zip';
+
+// This sets up a fetcher function, which will download a given URL
+// and return it as a blob.
+var fetcherConfig = {
+    id: 'fetch',
+    type: 'fetcher',
+    params: {
+    }
+};
+var fetcher = mldb.createFunction(fetcherConfig);
+
+//var res = mldb.query("SELECT tf_DecodeJpeg(fetch({url: 'file://mldb/ext/tensorflow/tensorflow/examples/label_image/data/grace_hopper.jpg'})[content], {ratio: 8}) as px");
```

Comment:
Did you mean to comment this? The log below will display the result of the previous query?